sprokit/src/sprokit/pipeline/pipeline.cxx

```diff
@@ -1344,16 +1351,17 @@ ::check_connection_flags(process::connection_t const& connection,
   {
     process::port_addr_t const& up_addr = connection.first;
 
-    shared_port_map_t::const_iterator const i = connected_shared_ports.find(up_addr);
+    const shared_port_map_t::const_iterator i = connected_shared_ports.find(up_addr);
```

Comment:
Question to satisfy my ignorance .. is the const here referring to the iterator itself?

Reply:
Yes. (Also, uh, please don't move the `const`; [[types.qualified]](../blob/master/doc/code-style.rst))